flink-python/docs/conf.py

```diff
@@ -102,6 +103,11 @@
 autosummary_generate = True
 autodoc_docstring_signature = True
 
+# Map to external docs for type annotations
+intersphinx_mapping = {
+    "pandas": ("https://pandas.pydata.org/docs/", None),
```

Comment:
I see [intersphinx docs](https://docs.readthedocs.com/platform/stable/guides/intersphinx.html) recommends 
`intersphinx_disabled_reftypes = ["*"]`. 
Should we add that too?

Reply:
I'm not sure that would work for us in this case - this pattern defines the non-external reftypes that intersphinx should not resolve to a mapping. The type annotations aren't themselves external refs, they're counted as internal ones so this disables the cross-referencing to the pandas docs.